cli/cmd/initnodes.go

```diff
@@ -398,3 +418,35 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 
 	return nil
 }
+
+func downloadAndRestoreLatestSnapshot(ctx context.Context, network string, clientName string) error {
```

Comment:
avoid using untyped params, rather pass in proper `network.ID`

Reply:
nit: use shorted names, suggest `downloadSnapshot`

Reply:
Done.